### PR TITLE
Add more Control/AxHost tests

### PR DESCRIPTION
**Pull request by hughbe**
## Proposed changes

- Add more Control/AxHost tests
- Focused around the open PRs right now to get a little more coverage

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/1813)

### CODE REVIEW

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PanelTests.cs [hunk 1]
```diff
@@ -101,8 +103,8 @@ public void Control_AutoSize_Set_GetReturnsExpected(bool value)
             Assert.Equal(value, control.AutoSize);
 
             // Set different.
-            control.AutoSize = value;
-            Assert.Equal(value, control.AutoSize);
+            control.AutoSize = !value;
```
Comment (RussKie) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PanelTests.cs]: Why has this changed?
I see it happened in few other places as well. I'd like to understand why we are making this change

Comment (hughbe) [reply src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PanelTests.cs]: This was a pasto. The intention of the test was to set it to a boolean value and then test setting the property to a different boolean value (i.e. `!value`)

So I fixed this pasto by updating it across the board

**Comment by RussKie** [comment]
What's a "pasto"? A similar to "typo"?

**Comment by hughbe** [comment]
> What's a "pasto"? A similar to "typo"?

When you copy-and-paste something and forget to change the new text!

**Pull request merged**